Assets/Scripts/Unit/Unit.cs

```diff
@@ -5,9 +5,26 @@
 using UnityEngine;
 using UnityEngine.Events;
 
+[Serializable]
 public class Unit
 {
-    ObjectIdentity objectIdentity;
-    public ObjectIdentity ObjectIdentity => objectIdentity;
-    // TODO: Add unit specific data here
-}
+    [SerializeField]
+    IdentityInfo identityInfo;
```

Comment:
Testing "Make a Suggestion" feature
```suggestion
    [SerializeField] IdentityInfo identityInfo;
    
```